tidy3d/web/api/webapi.py

```diff
@@ -366,6 +367,19 @@ def monitor(task_id: TaskId, verbose: bool = True) -> None:
         # Wait for task to finish
         prev_status = "draft"
         status = get_status(task_id)
+        if verbose:
+            console.log(
+                "To cancel the simulation, use 'web.abort(task_id)' or 'web.delete(task_id)' "
+                "or abort/delete the task in the web "
+                "UI. Terminating the Python script will not stop the job running on the cloud."
+            )
+            est_flex_unit = task_info.estFlexUnit
+            if est_flex_unit is not None and est_flex_unit > 0:
+                console.log(
+                    f"Maximum FlexCredit cost: {est_flex_unit:1.3f}. Use 'web.real_cost(task_id)'"
+                    f" to get the billed FlexCredit cost after a simulation run."
+                )
+
```

Comment:
Is this exactly the same code as in `FDTD`? If so let's put it in a separate function to call in both places.